src/search_type/symmetric_ledger.py

```diff
@@ -143,7 +141,7 @@ def collate_results(hits, entries, count=5):
         in hits[0:count]]
 
 
-def setup(config: TextSearchConfig, regenerate: bool) -> TextSearchModel:
+def setup(config: TextSearchConfigModel, regenerate: bool, verbose: bool) -> TextSearchModel:
```

Comment:
Let's drop `Model` from classes with names ending with `ConfigModel` now that we've removed `TextSearchConfig` and `ImageSearchConfig`